strongbox-data-service/src/main/java/org/carlspring/strongbox/config/ConnectionConfigOrientDB.java

```diff
@@ -116,26 +115,18 @@ public void setPassword(String password)
         this.password = password;
     }
 
-    public static String resolveProtocol(Environment environment)
+    public static String resolveProfile(Environment environment)
     {
-        return environment.getProperty(PROPERTY_PROTOCOL, "memory");
+        return environment.getProperty(PROPERTY_PROFILE, PROFILE_MEMORY);
     }
 
-    public static void bootstrap(String profile)
+    public static void bootstrap()
         throws IOException
     {
-        if (System.getProperty(PROPERTY_PROFILE) != null)
-        {
-            LOGGER.info(String.format("Can't override already provided OrientDB connection profile, skip profile [%s] bootstrap.",
-                                      profile));
-            return;
-        }
-
+        String profile = System.getProperty(PROPERTY_PROFILE, PROFILE_MEMORY);
         LOGGER.info(String.format("Bootstrap OrientDB connection properties with profile [%s].",
```

Comment:
Also, the names of loggers are all lowercase in our project (`logger`) everywhere across the codebase.

Reply:
done